ExtraEnemyCustomization/Customizations/EnemyAbilities/Abilities/EMP/Handlers/EMPSentryHandler.cs

```diff
@@ -0,0 +1,65 @@
+﻿using AK;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using UnityEngine;
+
+namespace EECustom.Customizations.EnemyAbilities.Abilities.EMP.Handlers
+{
+    public class EMPSentryHandler : EMPHandlerBase
+    {
+        static Color _offColor = new() { r = 0, g = 0, b = 0, a = 0 };
+        SentryGunInstance _sentry;
+        SentryGunInstance_ScannerVisuals_Plane _visuals;
```

Comment:
Protection definition is missing, inconsistent with rest of codebase.

```suggestion
        private static Color _offColor = new() { r = 0, g = 0, b = 0, a = 0 };
        private SentryGunInstance _sentry;
        private SentryGunInstance_ScannerVisuals_Plane _visuals;
```